src/sklearn_raster/estimator.py

```diff
@@ -216,6 +217,7 @@ def predict(
             allow_cast=allow_cast,
             check_output_for_nodata=check_output_for_nodata,
             nan_fill=0.0,
+            set_attrs={"source_method": self._method_name(wrapped_func)},
```

Comment:
I'm guessing you probably want to remain explicit here, which seems totally reasonable, but a helper function to set "set_attrs" could be used to reduce duplication?
```
def _set_attrs(self, func):
    return {"source_method": self._method_name(func)}
```
and
```
set_attrs=self._set_attrs(wrapped_func)
```
Probably only useful if you foresee adding any more attrs?

Reply:
I liked this suggestion, but I ended up removing this code when I switched to using the `history` attribute.